library/sql.inc

```diff
@@ -21,8 +21,8 @@
 */
 
 require_once(dirname(__FILE__) . "/sqlconf.php");
-require_once(dirname(__FILE__) . "/adodb/adodb.inc.php");
-require_once(dirname(__FILE__) . "/adodb/drivers/adodb-mysqli.inc.php");
+require_once("{$GLOBALS['vendor_dir']}/adodb/adodb-php/adodb.inc.php");
+require_once("{$GLOBALS['vendor_dir']}/adodb/adodb-php/drivers/adodb-mysqli.inc.php");
```

Comment:
[http://adodb.org/dokuwiki/doku.php?id=project:include_file_guide](http://adodb.org/dokuwiki/doku.php?id=project:include_file_guide)

> There is no support in ADOdb Version 5 for autoloading.
 There is no support in ADOdb Version 5 for namespacing.

Not sure the reasons, but they don't support it.